jena-fuseki2/jena-fuseki-ui/src/views/dataset/Edit.vue

```diff
@@ -237,13 +238,19 @@ export default {
       this.content = newVal
       this.codemirrorEditor.scrollTo(scrollInfo.left, scrollInfo.top)
     },
-    listCurrentGraphs: async function () {
+    listCurrentGraphs: async function (count=true) {
       this.loadingGraphs = true
       this.loadingGraph = true
       this.code = ''
       this.selectedGraph = ''
       try {
-        this.graphs = await this.$fusekiService.countGraphsTriples(this.datasetName, this.services.query['srv.endpoints'][0])
+        const endpoint = this.services.query['srv.endpoints'][0]
```

Comment:
#1679 has some changes for the `FusekiService` and the `src.endpoints`. Once that one is merged, I will rebase here and update the code to that calls `$fusekiService` methods.